src/org/mozilla/javascript/InterfaceAdapter.java

```diff
@@ -152,9 +152,8 @@ Object invokeImpl(
                 }
             }
         }
-        Scriptable thisObj = wf.wrapAsJavaObject(cx, topScope, thisObject, null);
```

Comment:
`thisObject` is an instance of `JavaAdapterInvokeTest.AdapterClass` and we want to call a method on `target`